catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/NewSpriteDialogFragment.kt

```diff
@@ -128,6 +119,53 @@ class NewSpriteDialogFragment(
             ?.create() as AlertDialog
     }
 
+    private fun handlePositiveButton(textInput: String?, currentScene: Scene) {
+        if (isObject) {
+            sprite = importProjectHelper!!.addObjectDataToNewSprite(null)
```

Comment:
Please use the nullable operator `?` instead of the not null assertion operator `!!`. I know this was just copied and the error already existed. For reference have a look at [https://medium.com/swlh/12-steps-to-convert-your-java-class-to-kotlin-the-right-way-9a718cfb498d](url)  _Step 4_

Therefore change the Sprite Member variable to a nullable type: `private var sprite: Sprite? = null`